src/geometry/PolySet.h

```diff
@@ -33,6 +33,7 @@ class PolySet : public Geometry
   void reserve(size_t numFacets) { polygons.reserve(numFacets); }
   void append_poly();
   void append_poly(const Polygon& poly);
+  void append_poly(bool marked);
```

Comment:
This the third overload of append_poly(), which is already not a great function. Consider refactoring into a single function.